### PR TITLE
add interfacce of ChainGovernance Precompiled Contract

### PR DESCRIPTION
**Pull request by sulenn**

### CODE REVIEW

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/config/system_config_service.go  3
- precompiled/utils.go  5
- precompiled/chaingovernance/chain_governance_service.go  15
- precompiled/chaingovernance/chain_governance_test.go  5
         

Clones added
============
- precompiled/config/system_config_service.go  2
- precompiled/chaingovernance/chain_governance_service.go  2
- precompiled/chaingovernance/chain_governance_test.go  3
- precompiled/consensus/consensus_service_test.go  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5867514&bid=19204362)

File: precompiled/chaingovernance/chain_governance_service.go [hunk 1]
```diff
@@ -0,0 +1,282 @@
+package chaingovernance
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/FISCO-BCOS/go-sdk/precompiled"
+
```
Comment (bxq2011hust) [inline precompiled/chaingovernance/chain_governance_service.go:9]: delete empty line

File: precompiled/chaingovernance/chain_governance_service.go [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+	operatorNotExist                int64 = -52007
+	operatorExist                   int64 = -52006
+	committeeMemberCannotBeOperator int64 = -52005
+	operatorCannotBeCommitteeMember int64 = -52004
+	invalidThreshold                int64 = -52003
+	invalidRequestPermissionDenied  int64 = -52002
+	committeeMemberNotExist         int64 = -52001
+	committeeMemberExist            int64 = -52000
+)
+
```
Comment (bxq2011hust) [inline precompiled/chaingovernance/chain_governance_service.go:33]:
```suggestion
type AccountInfo struct {
	Address   string `json:"address"`
	EnableNum string `json:"enable_num"`
}
```

File: precompiled/chaingovernance/chain_governance_service.go [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+	case committeeMemberExist:
+		message = "committee memberExist"
+	default:
+		message = ""
+	}
+	return message
+}
+
+// judgeErrorCode judges whether the error code represents an error
+func judgeErrorCode(errorCode int64) error {
```
Comment (bxq2011hust) [inline precompiled/chaingovernance/chain_governance_service.go:71]: ErrorCodeToError

File: precompiled/utils.go [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+	return message
+}
+
+func ParseBigIntFromOutput(abiStr, name string, receipt *types.Receipt) (*big.Int, error) {
+	parsed, err := abi.JSON(strings.NewReader(abiStr))
+	if err != nil {
+		fmt.Printf("parse ABI failed, err: %v", err)
+	}
+	var ret *big.Int
+	if len(receipt.Output) != 66 {
```
Comment (bxq2011hust) [inline precompiled/utils.go:43]: make 66 a const vat or delete this

File: precompiled/utils.go [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+// common precompiled contract error code
+const (
+	noAuthorized int64 = -50000
+)
+
+// non-precompiled contract error code, only used when logic errors occur
+const (
+	GoSDKErrorCode int64 = -1
```
Comment (bxq2011hust) [inline precompiled/utils.go:22]:
```suggestion
	DefaultErrorCode int64 = -1
```

File: precompiled/chaingovernance/account_info.go [hunk 6]
```diff
@@ -0,0 +1,6 @@
+package chaingovernance
```
Comment (bxq2011hust) [inline precompiled/chaingovernance/account_info.go:1]: delete this file

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/config/system_config_service.go  3
- precompiled/chaingovernance/chain_governance_service.go  15
- precompiled/utils.go  5
- precompiled/chaingovernance/chain_governance_test.go  5
         

Clones added
============
- precompiled/config/system_config_service.go  2
- precompiled/chaingovernance/chain_governance_service.go  2
- precompiled/consensus/consensus_service_test.go  2
- precompiled/chaingovernance/chain_governance_test.go  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5867514&bid=19204362)

**Pull request merged**